client/src/components/Verejne/entityHelpers.js

```diff
@@ -1,15 +1,10 @@
 // @flow
 import type {Entity} from '../../state'
 
-// entity which is a number is individual, district starts with letter
-export const isIndividual = (eid: string): boolean => {
-  return '0123456789'.indexOf(eid[0]) === -1
-}
-
```

Comment:
This function did not work as intended (the condition should be reversed) but nobody noticed because in reality `eid` is a number and `eid[0]` is `undefined`, so it always returned true ...

I have removed it completely because I couldn't find any evidence of current APIs returning string "district" eids. I believe the function is obsolete (but I'm not 100% sure, thus this comment ...)

Reply:
Note: The string eids were used to 'super entities' such as districts or cities - current APIs will not return such stuff.